features/bootstrap/FeatureContext.php

```diff
@@ -623,12 +626,16 @@ public function thereIsARelatedDummyWithFriends(int $nb)
         $relatedDummy = new RelatedDummy();
         $relatedDummy->setName('RelatedDummy with friends');
         $this->manager->persist($relatedDummy);
+        $this->manager->flush();
 
         for ($i = 1; $i <= $nb; ++$i) {
             $friend = new DummyFriend();
             $friend->setName('Friend-'.$i);
 
             $this->manager->persist($friend);
+            // since doctrine 2.6 we need existing identifiers on relations
+            // See https://github.com/doctrine/doctrine2/pull/6701
```

Comment:
Oh nice, you made my day! It's the solution to my problem on another project!

Reply:
:) happy xmas